README.md

```diff
@@ -270,8 +270,9 @@ OPTIONS
 
   -s, --tls
       Enable TLS encryption.
-                           Note that for kubernetes 'che-tls' with TLS certificate must be created in the configured 
-      namespace.
+                           Note, that this option is turned on by default for kubernetes infrastructure.
```

Comment:
So when this is on by default, how do you actually switch it off? Do we have something like `--no-tls`?

Reply:
No we don't have `--no-tls` flag, so a user cannot switch it off on Kubernetes-like infrastructures. The same is planned for Openshift soon. Although, we are going to keep the `--tls` flag for backward compatibility only.
Turning it off not only dangerous from user security point of view, but also some components might not work properly without TLS (as you probably know the story about Theia webviews).

Reply:
so if TLS is on by default and no longer disableable, the flag would effectively do nothing? 

Will you also ensure the fields in custom resource.yaml no longer work, so that it's no longer possible to set `tlsSupport: false` ?

Reply:
@nickboldt yes the flag will do nothing... Please see the thread https://github.com/che-incubator/chectl/pull/476#discussion_r373383036.

Reply:
As about `tlsSupport` flag in CRDs... you are right, we have to take a look and prevent breaking stuff with wrong value of the flag.

Reply:
But I think it should be done when I'll be working on operator part.

Reply:
OK as long as this TODO is captured in a GH issue for that task, then +1. Just don't want to forget this part. :D